plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/NavigatorViewBase.java

```diff
@@ -345,4 +356,73 @@ protected void redrawTree() {
         tree.getViewer().refresh();
     }
 
+    private void addAccessibilityListener(@NotNull DatabaseNavigatorTree navigatorTree) {
+        Tree tree = navigatorTree.getViewer().getTree();
+        if (tree == null) {
+            return;
+        }
+        Accessible accessible = tree.getAccessible();
+        accessible.addAccessibleListener(new AccessibleAdapter() {
+            @Override
+            public void getName(AccessibleEvent e) {
+                TreeViewer viewer = navigatorTree.getViewer();
+                IStructuredSelection selection = (IStructuredSelection) viewer.getSelection();
+                if (selection.size() > 1) {
+                    return;
+                }
+                Object firstElement = selection.getFirstElement();
```

Comment:
Why can't we use information from DBXTreeItem. We have item time (even plural) for every DBNDatabaseNode.
It doesn't make sense to use else/if and additional localizations

Reply:
Ok, changed